src/pages/blog/starlingx-dual-stack-support.md

```diff
@@ -0,0 +1,267 @@
+- The primary address pool family of a network cannot be modified after network creation, requiring a system reinstall to change.
+
+## API/CLI Interactions
+
+- The `network-addrpool-assign`, `network-addrpool-remove`, `network-addrpool-list`, and `network-addrpool-show` APIs handle network to address pool associations.
+- The `system network-add` command creates the first association.
+- The `system addrpool-modify` command allows for editing all address pool parameters.
+
+## Installation Bootstrap
+
+- Dual-stack installations require specifying secondary subnets in bootstrap variables using comma-separated values. E.g.:
+  - `external_oam_subnet: fd00::/64,10.20.2.0/24`, the order of network listing determines primary and secondary address pools. In the example fd00::/64 is the primary
+- All primary subnets must use the same address family.
+
+## DNS Server Configuration
+- Optionally, name servers can be configured with both IPv4 and IPv6 addresses using the `system dns-modify` command, still limited to 2 server addresses.
+
+## DC Operations
+- Subclouds can be installed in dual-stack mode if their version supports it.
+- All operational communication between the system controller and subclouds uses the primary address pool.
+- The system-controller and subclouds can operate in different network modes but must share the same primary address family in the OAM and management networks (for subclouds this can be OAM and admin networks).
+- Geo redundancy uses the primary pools to communicate.
+
+## Public Endpoints
+- Public OAM endpoints can be accessed via the secondary address using the same L4 port. HAProxy maps external requests to internal endpoints.
+
+## Network Address Modification at Runtime
+- Only the OAM, Admin, and Management networks can be modified using the `addrpool-modify` command during runtime. 
+- Other network’s address pool modifications require reinstallation.
+
+## External OAM API and CLI
+- The external-OAM API, although deprecated, can still be used to modify the OAM network primary pool. 
+- Corresponding CLIs are `system oam-modify` and `system oam-show`.
+
+## Kubernetes in Dual-Stack
+- The OAM, cluster-host, cluster-service, and cluster-pod networks must be configured for dual-stack support in Kubernetes. 
+- Runtime changes trigger quick restarts for the kube-API-server and kube-controller-manager pods. 
+- New pods will automatically receive both primary and secondary addresses when created in dual-stack mode. Existing pods may retain their current primary addresses and require restarts to acquire secondary addresses. 
+
+## Runtime Configuration
+- To add dual-stack to a running system, follow the steps outlined in the guide, which include adding address pools and associating them with networks.
+- Reverting to single-stack involves removing the network association with the address pool.
+
+# Configuring a Kubernetes network with dual-stack
```

Comment:
This is where I would move the second half of the current intro section. You can refer to it as a few things a user should know before starting to configure the dual-stack feature. Then I would give a brief overview on what configuration options you will go through, what the starting point is going to be and what people are expected to learn by the end of this article.

Reply:
done